modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1252 @@
+// bforte this is related to https://ephocks.atlassian.net/browse/DOC-1907
+////
+To quote my [comment, left on the TinyMCE 6.4 BoM][11] in Notion
+
+> What changes were made to the Advanced Code Premium plugin? The
+> [changelog][12] contains the below:
+> New button to copy the current dialog content. #TINY-9525
+> New buttons to increase and decrease the font size in the dialog. #TINY-9526
+> New button to toggle dark mode. #TINY-9527
+> New button to toggle fullscreen. #TINY-9529
+
+[14]: <https://github.com/tinymce/advcode/blob/main/CHANGELOG.md>
+
+[15]: <https://github.com/tinymce/advcode/commits/main>
+////
+
+[[accompanying-premium-skins-and-icon-packs-changes]]
+== Accompanying Premium Skins and Icon Packs changes
+
+The {productname} 6.4 release includes an accompanying release of the **Premium Skins and Icon Packs**.
+
+=== Premium Skins and Icon Packs
+
+The **Premium Skins and Icon Packs** release includes the following updates:
+
+The **Premium Skins and Icon Packs** were rebuilt to pull in the changes also incorporated into the default {productname} 6.4 skin, Oxide.
+
+For information on using premium skins and icon packs, see: xref:premium-skins-and-icons.adoc[Premium Skins and Icon Packs].
+
+
+[[improvement]]
+== Improvement
+
+{productname} 6.4 also includes the following improvement:
+
+=== Direct invalid child text nodes of list elements will be wrapped in list item elements.
+
+// https://ephocks.atlassian.net/browse/TINY-4818
+
+Invalid child nodes were throwing a `Uncaught TypeError:` when `<ol>` tags directly included a text node. This `TypeError` would not allow the user to `Tab` a text node while nested within `<ol>/<li>` HTML tags.
+
+Improvements were made to add a content filter for lists within {productname} 6.4. This improvement meant that any _invalid child nodes_ nested within `<li>` elements would be wrapped with this filter to avoid the `Uncaught TypeError:`.
+
+In {productname} 6.4, direct _invalid child text nodes_ of list elements are now wrapped in list item elements, which allow the user to now `Tab` nested text nodes that are within `<ol>/<li>` tags.
+// https://notion.so/tinycloud/Four-words-to-live-by-96e307110aaa4137b94f9ba12874d480
+
+=== Templates will be parsed before preview and insertion to make preview consistent with inserted template content and prevent XSS
+
+// https://ephocks.atlassian.net/browse/TINY-9244
+In previous versions of {productname}, templates were being parsed before `preview` and `insertion` to make preview consistent with content being inserted into the template to prevent Cross-Site Scripting (XSS) issues.
+As a consequence, the Templates plugin would not automatically format or sanitise the HTML passed to it as templates. This means that when previewing templates, the content may look different compared to when it is actually inserted into the editor. This can be a security issue, as unsanitised HTML can allow for Cross-Site Scripting (XSS), which can be harmful.
```

Comment:
```suggestion
As a consequence, the Templates plugin would not automatically format or sanitise the HTML passed to it as templates. This means that when previewing templates, the content may look different compared to when it is actually inserted into the editor. This can be a security issue, as unsanitized HTML can allow for Cross-Site Scripting (XSS), which can be harmful.
```